lib/demos/reference/motion_demo_shared_y_axis_transition.dart

```diff
@@ -0,0 +1,167 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN sharedYAxisTransitionDemo
```

Comment:
This can be removed I think

Reply:
Its needed for the code segments generator

Reply:
got it